src/types.ts

```diff
@@ -0,0 +1,28 @@
+export interface Todos {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+}
```

Comment:
The `Todos` interface is identical to the `Todo` interface except for the additional properties in `Todo`. Consider removing the `Todos` interface or renaming it to avoid redundancy and confusion.